src/Zicht/Bundle/SolrBundle/Resources/config/services.xml

```diff
@@ -48,7 +48,7 @@
         <service id="zicht_solr.solr" class="%zicht_solr.solr.class%">
             <argument type="service" id="zicht_solr.http_client"/>
             <argument>%zicht_solr.absolute_base_url%</argument>
-            <argument>%solr_core%</argument>
+            <argument>%env(resolve:SOLR_CORE)%</argument>
```

Comment:
Dit kun je ook lokaal fixen in je parameters. Dan blijft de bundle vrij van `env`

```yaml
# config/packages/zicht_solr.yaml
parameters:
    solr_core: '%env(resolve:SOLR_CORE)%'
```

Ofzoiets :grin: